Sources/MuxUploadSDK/PublicAPI/MuxUpload.swift

```diff
@@ -475,6 +481,19 @@ public final class MuxUpload : Hashable, Equatable {
         if !uploadInfo.options.inputStandardization.isEnabled {
             startNetworkTransport(videoFile: videoFile)
         } else {
+            let inputStandardizationStartTime = Date()
+            let reporter = Reporter.shared
+
+            // For consistency report non-std
```

Comment:
Retrospectively, now think we should report the standardized input size if we can. I'll update this PR with that change.